docs/theming_plone_5/custom-components.md

```diff
@@ -35,8 +35,7 @@ Plone has default Viewlets and ViewletManagers like ContentAbove and BelowConten
 
 For an overview of existing Viewlets and ViewletManagers look at the `/@@manage-viewlets` view.
 
-For more details about Viewlets/ViewletManagers and there possibilities see the Viewlets sections of the
-[Mastering Plone 5: Development chapters](https://training.plone.org/5/mastering-plone/viewlets_1.html).
+For more details about Viewlets/ViewletManagers and their possibilities, see {doc}`mastering-plone/viewlets_1`.
```

Comment:
My mistake, I forgot to make the path root-relative. However can you check this by making a local build? That will save review cycles. If not, then you can always check the Netlify preview build. For example, the latest build shows that this link did not render: https://deploy-preview-742--plone-training.netlify.app/theming_plone_5/custom-components#viewlets

```suggestion
For more details about Viewlets/ViewletManagers and their possibilities, see {doc}`/mastering-plone/viewlets_1`.
```

Reply:
I'll definitely keep this mind from next time, now i have updated the file and the link do renders in Netlify preview build